src/components/database/EditDatabaseDialog.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+    <v-dialog v-model="dialogOpen" max-width="600">
+        <v-unipept-card>
+            <v-card-title class="text-h6 font-weight-bold">
+                Confirm Update
+            </v-card-title>
+            <v-card-text>
+                <p>
+                    Are you sure you want to update this custom database <strong>{{ database }}</strong>?
+                    This action is irreversible.
+                </p>
+                <v-alert type="warning" class="mt-4">
+                    Updating this database will trigger a <em>reanalysis</em> of all samples that currently use this database.
+                    This reanalysis will be performed using the most recent version of the underlying UniProtKB database.
```

Comment:
Do you think it would be hard to only show this warning when there are actually samples linked to this database?